src/Components/Articles.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState, useEffect } from "react";
+import { getArticles } from "./Api";
+import { Link } from "react-router-dom";
+
+export default function () {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articles) => {
+      setArticles(articles);
+    });
+  }, []);
+
+  const extractSummary = (body, maxLength) => {
+    const summary = body.substring(0, maxLength);
+    return body.length > maxLength ? `${summary}...` : summary;
+  };
+
+  return (
+    <div className="articles">
+      <h2>Explore articles</h2>
+      <ul>
+        {articles.map((article) => {
+          return (
+            <li key={article.article_id}>
+              <img src={article.article_img_url} alt="Article Image" />
+              <h2>{article.title}</h2>
+              <p>
+                <strong>Topic:</strong> {article.topic}
+              </p>
+              <p>
+                <strong>Author:</strong> {article.author}
+              </p>
+              <p>
+                <strong>Publication Date:</strong>{" "}
+                {new Date(article.created_at).toLocaleDateString()}
+              </p>
+              <p>
+                <strong>Comments:</strong> {article.comment_count}
+              </p>
+              <p>{extractSummary(article.body, 100)}</p>
+              <Link
+                to={`/articles/${article.article_id}`}
+                className="read-more-link"
+              >
+                Read more
+              </Link>
+            </li>
```

Comment:
Please de-bloat this component by extracting this code into a sub-component. Normally ArticleCard is what we'd call a component like this but you can call it whatever you want. We want to do this because then you have a reusable component that renders a article that will go in a list. You can for example reuse this component for your recentArticles view in the home component too. This will de-bloat two files with a reusable component.